index.html

```diff
@@ -128,127 +130,62 @@ <h2 class="heading2 multy-task">Multi-Post Stories</h2>
                     </li>
                 </ul>
                 <form>
-                    <button type="submit" class="animate-btn project-desktop-button button1 button1-text">See
+                    <button type="submit" class="animate-btn project-desktop-button button1 button1-text btn-active">See
```

Comment:
- [ ] As suggested previously, kindly display the popup for all cards. Currently, the popup is not displayed for Multi post stories as shown below. Fix it.
- [ ] As suggested previously to make all cards dynamically from js. Kindly create Multi post stories dynamically from js and remove it from HTML.

https://user-images.githubusercontent.com/70269507/174431567-4b5986cf-0e41-459f-885f-45986630fb90.mov